custom/enikshay/const.py

```diff
@@ -105,7 +105,9 @@
     ("address_line_1", "Address Line 1", []),
     ("address_line_2", "Address Line 2", []),
     ("pincode", "Pincode", []),
-    ("mobile_no_1", "Mobile No. 1", []),
+    # TODO add format validation (91 + 10 digits, no leading zeros)
+    ("contact_phone_number", "Mobile number for SMS alerts (please only enter 91 and ten digits, not beginning with 0)", []),
```

Comment:
maybe we should say:

Please enter only digits. Enter 91 followed by the 10-digit national number.